h2integrate/finances/profast_financial.py

```diff
@@ -582,9 +622,28 @@ def compute(self, inputs, outputs):
                 tech_opex_info.setdefault(fix_cost_key, fix_cost_val)
             fixed_costs[tech] = tech_opex_info
 
+            # get tech-specific variable cost parameters
+            tech_varopex_info = (
+                self.tech_config[tech]["model_inputs"]
+                .get("financial_parameters", {})
+                .get("variable_costs", {})
+            )
+
+            # add CapEx cost to tech-specific variable cost entry
+            varom_cost_per_unit_commodity = inputs[f"varopex_adjusted_{tech}"] / total_production
+            if not all(v == 0.0 for v in varom_cost_per_unit_commodity):
+                varom_dict = dict(zip(years_of_operation, varom_cost_per_unit_commodity))
+                tech_varopex_info.update({"cost": varom_dict})
```

Comment:
```suggestion
            # add CapEx cost to tech-specific variable cost entry
            varopex_adjusted_tech = inputs[f"varopex_adjusted_{tech}"]
            if np.any(varopex_adjusted_tech) > 0:
                varopex_cost_per_unit_commodity = varopex_adjusted_tech / total_production
                varopex_dict = dict(zip(years_of_operation, varopex_cost_per_unit_commodity))
                tech_varopex_info.update({"cost": varopex_dict})
```

Two main changes here:
- using numpy array logic for the if statement to make it easier to read
- using `varopex` instead of `varom` to be consistent naming-wise